os/windows/installer/version_win32.txt

```diff
@@ -1,5 +1,5 @@
 !define APPBITS 32          ; Define number of bits for the architecture
-!define EXTRA_VERSION "XP SP3, Vista and 7"
+!define EXTRA_VERSION "XP SP3 and Newer"
```

Comment:
Seems like "Newer" should not be capitalized here.

Reply:
Resolved in f6dcd85